helix-lsp/src/client.rs

```diff
@@ -201,6 +202,7 @@ impl Client {
         let (server_rx, server_tx, initialize_notify) =
             Transport::start(reader, writer, stderr, id);
         let (workspace, workspace_is_cwd) = find_workspace();
+        let workspace = path::get_normalized_path(&workspace);
```

Comment:
What do you think about canonicalizing in `find_workspace` instead?

Reply:
I wanted to do that at first but the problem with that is the `normalize` is in `helix-core` and `find_workspace` is in `helix-loader` (which can't pull in `helix-core`). All other uses of `find_workspace` don't care about path equality (they just use it to do IO where it doesn't matter which path is used)